<a href="https://colab.research.google.com/github/alexandralima1602/-C8280-./blob/main/ACTIVIDAD1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
import pandas as pd
# Leer el archivo de datos
data = pd.read_csv("data.tsv", delimiter="\t")




In [ ]:
Datos = pd.read_table("data.tsv")
Datos

order_id  quantity                              item_name  \
0            1         1           Chips and Fresh Tomato Salsa   
1            1         1                                   Izze   
2            1         1                       Nantucket Nectar   
3            1         1  Chips and Tomatillo-Green Chili Salsa   
4            2         2                           Chicken Bowl   
...        ...       ...                                    ...   
4617      1833         1                          Steak Burrito   
4618      1833         1                          Steak Burrito   
4619      1834         1                     Chicken Salad Bowl   
4620      1834         1                     Chicken Salad Bowl   
4621      1834         1                     Chicken Salad Bowl   

                                     choice_description item_price  
0                                                   NaN     $2.39   
1                                          [Clementine]     $3.39   
2                                               [Apple]     $3.39   
3                                                   NaN     $2.39   
4     [Tomatillo-Red Chili Salsa (Hot), [Black Beans...    $16.98   
...                                                 ...        ...  
4617  [Fresh Tomato Salsa, [Rice, Black Beans, Sour ...    $11.75   
4618  [Fresh Tomato Salsa, [Rice, Sour Cream, Cheese...    $11.75   
4619  [Fresh Tomato Salsa, [Fajita Vegetables, Pinto...    $11.25   
4620  [Fresh Tomato Salsa, [Fajita Vegetables, Lettu...     $8.75   
4621  [Fresh Tomato Salsa, [Fajita Vegetables, Pinto...     $8.75   

[4622 rows x 5 columns]

**Mire en la parte superior e inferior de data.tsventregado en esta actividad. Piensa en cómo se estructuran los datos. ¿Qué crees que significa cada columna? ¿Qué crees que significa cada fila?**

In [ ]:
# Mostrar la parte superior de los datos (las primeras filas)
print(data.head())

   order_id  quantity                              item_name  \
0         1         1           Chips and Fresh Tomato Salsa   
1         1         1                                   Izze   
2         1         1                       Nantucket Nectar   
3         1         1  Chips and Tomatillo-Green Chili Salsa   
4         2         2                           Chicken Bowl   

                                  choice_description item_price  
0                                                NaN     $2.39   
1                                       [Clementine]     $3.39   
2                                            [Apple]     $3.39   
3                                                NaN     $2.39   
4  [Tomatillo-Red Chili Salsa (Hot), [Black Beans...    $16.98   


In [ ]:
#  parte inferior de los datos (las últimas filas)
print(data.tail())

      order_id  quantity           item_name  \
4617      1833         1       Steak Burrito   
4618      1833         1       Steak Burrito   
4619      1834         1  Chicken Salad Bowl   
4620      1834         1  Chicken Salad Bowl   
4621      1834         1  Chicken Salad Bowl   

                                     choice_description item_price  
4617  [Fresh Tomato Salsa, [Rice, Black Beans, Sour ...    $11.75   
4618  [Fresh Tomato Salsa, [Rice, Sour Cream, Cheese...    $11.75   
4619  [Fresh Tomato Salsa, [Fajita Vegetables, Pinto...    $11.25   
4620  [Fresh Tomato Salsa, [Fajita Vegetables, Lettu...     $8.75   
4621  [Fresh Tomato Salsa, [Fajita Vegetables, Pinto...     $8.75   


RSPTA: Las columnas significan los parametros de toda la data como ID,	cantidad,	nombre del árticulo,	elección_descripción y el precio del articulo, las filas significan los ejemplos de los parametros

**¿Cuántos pedidos parece haber?**

In [ ]:
pedidos = data["quantity"].sum()
print( "pedidos:", pedidos)


pedidos: 4972


Viendo la suma de la cantidad de pedidos llegamos a la conclusiòn de que hay 4972 pedidos

**¿Qué burrito es más popular, steako chicken?**

In [ ]:

#Steak Burrito
steak_burrito = data["item_name"].value_counts()["Steak Burrito"]
print(" Steak Burrito:", steak_burrito)


 Steak Burrito: 368


In [ ]:
#Chicken_burrito
Chicken_burrito = data["item_name"].value_counts()["Chicken Burrito"]
print(" Chicken Burrito:", Chicken_burrito)


 Chicken Burrito: 553


El mas popular es Chiken Burrito por que tiene mayor cantidad de pedidos con 553

**¿cuantas lineas hay en este archivo?**

In [ ]:
len(data)

4622

Hay 4622 lineas



```
Esto tiene formato de código
```

**¿Los burritos de pollo suelen tener frijoles negros o frijoles pintos?**

In [ ]:
burritos_pollo = data[data["item_name"] == "Chicken Burrito"]



In [ ]:
# Contar la cantidad de burritos de pollo con frijoles negros y frijoles pintos
frijoles_negros = burritos_pollo[burritos_pollo["choice_description"].str.contains("Black Beans", case=False)].shape[0]
frijoles_pintos = burritos_pollo[burritos_pollo["choice_description"].str.contains("Pinto Beans", case=False)].shape[0]

In [ ]:
print("Burritos de pollo con frijoles negros:", frijoles_negros)
print("Burritos de pollo con frijoles pintos:", frijoles_pintos)


Burritos de pollo con frijoles negros: 282
Burritos de pollo con frijoles pintos: 105


Los burritos de pollo suelen tener frijoles negros como frijoles pintos

**Usa la línea de comandos para descubrir algo interesante sobre los datos dados.**

In [94]:
#Valor mínimo de los precios de articulos

precio_minimo = data['item_price'].min()

print("Valor mínimo ", precio_minimo)


Valor mínimo  $1.09 
